sheetal-dave17/crud-todo-app
# added storybook UI components

**Set up Storybook and Added few components**

What is a storybook?
- Storybook is an open-source tool for developing UI components in isolation for React, Vue, Angular, and more.  
It makes building stunning UIs organized and efficient  [readMore](https://storybook.js.org/docs/react/get-started/introduction)

**How To Test**
- checkout `storybook-setup` branch 
- do npm install
- run storybook using `npm run storybook`
- verify components are created correctly with different attributes


**Note: I have just created some reusable component regardless of logic and functionalities i.e: different types of buttons like for success, error, warning, etc.., created a list component will we used to display any listing of items, etc...**
